src/Samples/Sample.AspNetCore/Startup.cs

```diff
@@ -1,75 +1,105 @@
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+            else
+            {
+                app.UseExceptionHandler("/Home/Error");
+            }
+
+            // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+            app.UseHsts();
+            app.UseHttpsRedirection();
+            app.UseStaticFiles();
+            app.UseSession();
+            app.UseRouting();
+
+            app.UseAuthorization();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllerRoute(
+                    "default",
+                    "{controller=Products}/{action=Index}/{id?}");
+            });
+
+            Console.WriteLine("Test : Environment variables!");
+            Console.WriteLine("One time setup");
+            foreach (DictionaryEntry variable in Environment.GetEnvironmentVariables())
+            {
+                Console.WriteLine($"{variable.Key}: {variable.Value}");
+            }
+        }
+
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddDbContext<StoreDbContext>(options => options.UseInMemoryDatabase("Products"));
+            services.AddControllersWithViews();
+            services.AddDistributedMemoryCache();
+            Guid.TryParse(Configuration.GetSection("PayeeInfo")["PayeeId"], out Guid payeeId);
+            services.Configure<PayeeInfoConfig>(options =>
+            {
+                options.PayeeId = payeeId;
+                options.PayeeReference = DateTime.Now.Ticks.ToString();
+            });
+            services.Configure<UrlsOptions>(Configuration.GetSection("Urls"));
+            services.AddScoped(provider => SessionCart.GetCart(provider));
+            services.AddTransient<IHttpContextAccessor, HttpContextAccessor>();
+            services.AddSwedbankPayClient(Configuration);
+            services.AddSession();
+
+            // Code copied from:
+            // https://docs.microsoft.com/en-us/aspnet/core/security/enforcing-ssl?view=aspnetcore-3.1&tabs=visual-studio#options
+            services.AddHsts(options =>
+            {
+                options.Preload = true;
+                options.IncludeSubDomains = true;
+                options.MaxAge = TimeSpan.FromDays(60);
+            });
+
+            services.AddHttpsRedirection(options =>
+            {
+                options.RedirectStatusCode = StatusCodes.Status307TemporaryRedirect;
+                options.HttpsPort = 5001;
+            });
+
+            Console.WriteLine("Test : app config!");
+            Console.WriteLine(payeeId);
+            foreach(var url in Configuration.GetSection("Urls").GetChildren())
+            {
+                Console.WriteLine($"{url.Key}: {url.Value}");

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5523315)